README.md

```diff
@@ -38,6 +38,16 @@ curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain none
 See also [Other installation methods](
 https://github.com/rust-lang-nursery/rustup.rs/#other-installation-methods)
 
+### Clone project
+
+```sh
+# using https
+git clone https://github.com/servo/servo.git
+
+# using ssh
+git@github.com:servo/servo.git
+```
+
```

Comment:
I'm not against adding this but… this feels a bit unnecessary?

Reply:
Happy to remove! Your call

Reply:
Yeah let's remove it :)

Thanks!